docs/release_notes.md

```diff
@@ -0,0 +1,42 @@
+PC Engines APU2 Coreboot Release Notes
+--------------------------------------
+
+### 4.0.1
+
+#### Coreboot & SeaBIOS
+
+  * introduce versioning
+  * obtain `smbios` fields, such as `board serial` and `SKU`
+  * allow for one-time `PXE ` boot with `N` key
+  * reduce log level: display mainboard, DRAM and ECC info only
+  * improve SD card performance
+  * force to use SD in 2.0 mode
+  * prevent from printing character multiple times
+  * nct5104d driver backport
+  * check if `git` exists before calling it
+  * change git repository in `Makefile`
+  * add missing `cbfs_find_string` in `libpayload`
+  * add missing binaries
+  * add executable bit to `xcompile`
+  * U-ELTAN release 20160304
```

Comment:
@macpijan this line can be removed and note on top can be added that 4.0.x releases based on PC Engines 20160304 release.